src/components/switch/Switch.stories.mdx

```diff
@@ -133,7 +133,13 @@ There are three different sizes for switches: Small, Medium, and Large. The size
             innerWidth: { table: { disable: true } },
             className: { table: { disable: true } },
             onClick: { action: 'clicked' },
-            ...Utils.hideComponentProperties(['width', 'height', 'innerWidth', 'className', 'onClick']),
+            ...Utils.hideComponentProperties([
```

Comment:
we should now be able to precisely define in the component which props we want to make public ... we can still keep that helper for convenience though